coc-committee-description-proposed.md

```diff
@@ -1,58 +1,62 @@
 **PROPOSED DRAFT: This proposed working draft is under review and development by the [Code of Conduct Working Group](https://github.com/cncf/wg-coc). It is not a final document. If you would like to propose changes to this document, please do so by opening a pull request, opening an issue, or posting a comment. The first version of this proposed draft was contributed by the [Interim CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) and is based on the interim committee’s own [committee description](https://www.cncf.io/conduct/committee/).**
 
-# CNCF Code of Conduct Committee
+# CNCF Code of Conduct Committee Charter
 
 ## Purpose
 
 The CNCF Code of Conduct Committee (“CoC Committee”) responds to, investigates, and resolves [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) incidents.
 
-The Jurisdiction and Escalation Policy describes the CoC Committee’s relationship to the Linux Foundation (LF) events staff, who enforce the [Linux Foundation Events Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/), and project-level Code of Conduct responders (e.g., the Kubernetes Code of Conduct Committee).
+The Jurisdiction Policy describes the CoC Committee’s relationship to the Linux Foundation (LF) staff, who enforces the [Linux Foundation Events Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/), and project-level Code of Conduct responders (e.g., the Kubernetes Code of Conduct Committee).
 
-## Members
+## Composition and Selection of Committee Members
 
-The Interim CoC Committee consists of 3 community members, 1 LF Events staff member, and 2 CNCF staff members. The members of the interim committee are listed below:
+The CoC Committee consists of 3 community members and 2 CNCF staff members, plus alternates.
 
-**Community Members:**
+### Community Representatives
 
--   [Name 1]
--   [Name 2]
--   [Name 3]  
-      
-**Alternate Community Members:**
+The community representatives will be selected as follows:
 
--   [Name 1]
--   [Name 2]
--   [Name 3]  
-      
-**CNCF Staff:**
+* The [Technical Oversight Committee](https://www.cncf.io/people/technical-oversight-committee/) (TOC) will appoint 2 primary representatives and 2 alternate representatives, each of whom must be a current or emeritus member of the TOC.
+* [CNCF TAG Contributor Strategy](https://github.com/cncf/tag-contributor-strategy/) will appoint 1 primary representative and 1 alternate representative, each of whom must be a current member of TAG Contributor Strategy.
 
--   [Name 1]
--   [Name 2]
+There are no fixed terms or term limits. The TOC and TAG-Security may replace their designated representatives at any time. However, for continuity, it is highly encouraged that each representative serve for a duration of 1 year or longer.
```

Comment:
One suggestion would be that "After 3 events where a representative is unable to participate (excluding absences due to a conflict of interest), they are encouraged to step down so that another representative with more time to dedicate to the committee can be elected"

Reply:
Good suggestion, @carolynvs. I'll discuss with the working group co-chairs today